clang/lib/CConv/ConstraintBuilder.cpp

```diff
@@ -64,6 +66,7 @@ class FunctionVisitor : public RecursiveASTVisitor<FunctionVisitor> {
       }
       if (VarDecl *VD = dyn_cast<VarDecl>(D)) {
         if (VD->isLocalVarDecl()) {
+          Info.getABoundsInfo().insertVariable(VD);
```

Comment:
Why not something like this for global variables too? I don't see it in `VisitVarDecl`

Reply:
Done.